QRScanner/QRScannerView.swift

```diff
@@ -10,9 +10,9 @@ import UIKit
 import AVFoundation
 
 public protocol QRScannerViewDelegate: AnyObject {
-    func failure(_ error: QRScannerError)
-    func success(_ code: String)
-    func didChangeTorchActive(isOn: Bool)
+    func qrScannerView(_ qrScannerView: QRScannerView, didFailure error: QRScannerError)
+    func qrScannerView(_ qrScannerView: QRScannerView, didSuccess code: String)
+    func qrScannerView(_ qrScannerView: QRScannerView, didChangeTorchActive isOn: Bool)
```

Comment:
[nits] Add `optional` for this delegate func? When users don't use FlashButton, this func is unnecessary.

Reply:
I see!